example/lib/map_ui.dart

```diff
@@ -353,51 +366,42 @@ class MapUiBodyState extends State<MapUiBody> {
     );
 
     final List<Widget> columnChildren = <Widget>[
-      Padding(
-        padding: const EdgeInsets.all(10.0),
-        child: Center(
-          child: SizedBox(
-            width: 300.0,
-            height: 200.0,
-            child: mapboxMap,
-          ),
+      Center(
+        child: SizedBox(
+          width: _mapExpanded ? null : 300.0,
+          height: 200.0,
+          child: mapboxMap,
         ),
       ),
     ];
 
     if (mapController != null) {
-      columnChildren.add(
-        Expanded(
-          child: ListView(
-            children: <Widget>[
-              Text('camera bearing: ${_position!.bearing}'),
-              Text(
-                  'camera target: ${_position!.target.latitude.toStringAsFixed(4)},'
-                  '${_position!.target.longitude.toStringAsFixed(4)}'),
-              Text('camera zoom: ${_position!.zoom}'),
-              Text('camera tilt: ${_position!.tilt}'),
-              Text(_isMoving ? '(Camera moving)' : '(Camera idle)'),
-              _queryFilterToggler(),
-              _compassToggler(),
-              _myLocationTrackingModeCycler(),
-              _latLngBoundsToggler(),
-              _setStyleToSatellite(),
-              _zoomBoundsToggler(),
-              _rotateToggler(),
-              _scrollToggler(),
-              _tiltToggler(),
-              _zoomToggler(),
-              _myLocationToggler(),
-              _telemetryToggler(),
-              _visibleRegionGetter(),
-            ],
-          ),
-        ),
+      columnChildren.addAll(
+        <Widget>[
+          Text('camera bearing: ${_position.bearing}'),
+          Text('camera target: ${_position.target.latitude.toStringAsFixed(4)},'
+              '${_position.target.longitude.toStringAsFixed(4)}'),
+          Text('camera zoom: ${_position.zoom}'),
+          Text('camera tilt: ${_position.tilt}'),
+          Text(_isMoving ? '(Camera moving)' : '(Camera idle)'),
+          _mapSizeToggler(),
+          _queryFilterToggler(),
+          _compassToggler(),
+          _myLocationTrackingModeCycler(),
+          _latLngBoundsToggler(),
+          _setStyleToSatellite(),
+          _zoomBoundsToggler(),
+          _rotateToggler(),
+          _scrollToggler(),
+          _tiltToggler(),
+          _zoomToggler(),
+          _myLocationToggler(),
+          _telemetryToggler(),
+          _visibleRegionGetter(),
+        ],
       );
     }
-    return Column(
-      mainAxisAlignment: MainAxisAlignment.start,
-      crossAxisAlignment: CrossAxisAlignment.stretch,
+    return ListView(
       children: columnChildren,
```

Comment:
Nit: maybe also rename columnChildren, if we now use a ListView

Reply:
will do - as a side note i did this changes to make sure that turning off texture mode does not cause any issues with more dynamic maps.

Reply:
So the previous layout didn't work correctly with texture mode turned off?

Reply:
no idea - but the documentation of texture mode indicates to use it for more dynamic scenarios. So i added this to make sure that resizing and scrolling work fine

Reply:
Ah now I get it. I misunderstood your comment and thought you had seen issues with the previous layout and more dynamic maps.